CodePushConfig.m

```diff
@@ -31,6 +33,14 @@ - (instancetype)init
     NSString *deploymentKey = [infoDictionary objectForKey:@"CodePushDeploymentKey"];
     NSString *serverURL = [infoDictionary objectForKey:@"CodePushServerURL"];
     
+    NSUserDefaults* userDefaults = [NSUserDefaults standardUserDefaults];
+    NSString* clientUniqueId = [userDefaults stringForKey:ClientUniqueIDConfigKey];
```

Comment:
In what scenario would this value change that would make it beneficial for us to persist it?